src/bibliography/ui/BibliographySelect.tsx

```diff
@@ -52,6 +52,7 @@ export default class BibliographySelect extends Component<
     return (
       <>
         <AsyncSelect
+          aria-label="BibliographySelect"
```

Comment:
There is already a label defined on the next line.

Labels should be meaningful to the users, not component names. [data-testid](https://testing-library.com/docs/dom-testing-library/api-queries#bytestid) should be used if an ID is needed for testing, but it is seldom needed, e.g. the BibliohraphySelect has a label which can be used.

Reply:
Would it make sense to change the "aria-labelledby" to "aria-label" or why are we use "aria-labelledby" ?

Reply:
To tell which element is the label. If you change it `ReferenceForm` will get broken. If you absolutely have to go without a real label (can't remember if it was a requirement not have labels here) you can make it take either or.

In general, it is considered a bad practice not to have real labels. See e.g.:

- https://joshuawinn.com/ux-input-placeholders-are-not-labels/
- https://uxplanet.org/designing-more-efficient-forms-structure-inputs-labels-and-actions-e3a47007114f
- https://www.uxbooth.com/articles/the-new-rules-of-form-design/

Reply:
i decided to get rid of `aria-labelledyby' and use `aria-label` instead.
For `aria-labelled-by` there is a `label` (with text needed) but in `SearchForm.tsx` there is no need for a label with text. If the `label` is empty there is no way of accesing it in the test other than using the `labels id` which is bad practiced. That's why i got rid of `aria-labelled-by`. `aria-label` gets passed: `_.uniqueId('ReferenceForm-Document-')` which i access in the tests using regex `/ReferenceForm-Document-.*` via `changeValueByLabel`